services/QuillLMS/client/app/bundles/Teacher/components/subscriptions/SubscriptionStatus.jsx

```diff
@@ -62,6 +89,15 @@ const SubscriptionStatus = ({
     remainingDays = expiration.diff(moment(), 'days');
   }
 
+  const content = {
+    status: <h2>{`You have a ${subscriptionTypeText} subscription`}<img alt={`${subscriptionTypeText}`} src={`https://assets.quill.org/images/shared/${image}`} /></h2>,
+    buttonOrDate: expiration && (
+      <span className="expiration-date">
+        <span>Valid Until:</span> <span>{`${expiration.format('MMMM Do, YYYY')}`}</span><span className="time-left-in-days"> | {`${remainingDays} ${pluralize('days', remainingDays)}`}</span>
+      </span>
+    )
+  };
+
   switch (subscriptionType) {
```

Comment:
I think this is a huge improvement on the previous code which had too many conditionals.  Long term, we could possibly get rid of the switch statement with the pattern [Replace Conditional with Polymorphism](https://refactoring.guru/replace-conditional-with-polymorphism).